### PR TITLE
User manual/instructions

### LINKED ISSUE #136: Update description to ideal posture recomendation
# Goal
Identify a good posture guide for users to follow when picking their ideal pose

### LINKED ISSUE #146: User Manual for PosturePerfection
# Goal

Add a new section to the GitHub Pages outlining exactly how the system works for an end user

# Plan

- Use steps from help and about screen in app
- Add links to online posture guides
- Add instructions on using the receiver
- Put link to this manual in the project README

### PR DESCRIPTION
**Pull request by Ashwin-MJ**
# Details

- New page in GitHub Pages which provides instructions for how the system works
- Re-ordered pages

# Screenshots/Gif (if appropriate)

Probably best to run the docs on this branch locally to spot any issue

# Checklist

## Code
N/A

## Documentation
- [x] I have updated documentation appropriately (GitHub Pages)
- [x] I have tested the updated GitHub Pages locally to ensure the changes to documentation render correctly (see this [Guidance](https://ese-peasy.github.io/PosturePerfection/guidance.html) page)

Closes #146 #136

### CODE REVIEW

File: docs/instructions.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+nav_order: 5
+title: Instructions
+---
+
+# Instructions
```
Comment (miklasr) [inline docs/instructions.md]: Could we rename this to "usage instructions", "usage manual", or something else that indicates this is the manual. In the context of the rest of our Pages "Instructions" is a bit ambiguous.

File: docs/instructions.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+If pose estimation fails to detect your whole body, we bring this to your attention in the form of <span style="color:grey">Grey</span> lines - if you see this, please adjust your camera or your own positioning until you are clearly detectable in the video feed. In the Settings we have a slider for "Confidence Threshold" which you can tune to modify this - if you feel like you're being presented with an <span style="color:grey">"Unknown Posture"</span> even when you are in an optimal position, decrease this value a bit and that should improve your experience.
+
+<div align="center">
+    <img src="images/instructions/unknown_posture.png" width="500"/>
+</div>
+
+## Achieving PosturePerfection!
+
+When you are happy with a posture that you have adopted and have clicked "Set Ideal Posture", your detected posture will be highlighted in <span style="color:green">Green</span> as you are then in a position where you have achieved PosturePerfection! Carry on with your work, and the system will let you know when we detect you have strayed from this ideal posture. See below for more details to ensure you maximise your time in PosturePerfection!
```
Comment (miklasr) [inline docs/instructions.md]: "... ensure you ~maximise~ **optimise** your time in PosturePerfection!"

Make sure it doesn't sound like we want them to spend lots of time on the app. I assume you are saying we want them to get the most out of it.

File: docs/instructions.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+<div align="center">
+    <img src="images/instructions/bad_posture.png" width="500"/>
+</div>
+
+## Setting up Notifications
+
+We are aware that some of you may choose to setup PosturePerfection on a separate device from your work device. Therefore, we have incorporated the sending of notifications from PosturePerfection to another Linux device on the same network when the system detects you have started slouching! Note that this isn't an exclusive feature - you can even receive these notifications on the same device on which you are running PosturePerfection!
+
+To do this, ensure you have `simple-remote-notify-send` installed. Find the latest release and more information [here](https://github.com/ESE-Peasy/simple-remote-notify-send/releases/tag/v1.0.0). Run the `PosturePerfectionReceiver` on the device where you want to receive notifications, and then start `PosturePerfection`.
```
Comment (miklasr) [inline docs/instructions.md]: Could we make it completely clear that this is part of ***our*** software? It think that might make users more comfortable in using it and of course clarifies that for getting marks. (Possibly even add install instructions here, what do you think?)

**Review by miklasr — CHANGES_REQUESTED** [review]
Really good work. These are some super sleek usage instructions! I've made a few comments on things I think we should change. Do you want me to add a link from the GUI to this page in this PR?

File: docs/instructions.md (hunk 1 above)
Comment (Ashwin-MJ) [reply docs/instructions.md]: Sure thing, I had "Instructions" as a placeholder and was actually going to ask for better suggestions. Will change to "Usage Manual"

File: docs/instructions.md (hunk 3 above)
Comment (Ashwin-MJ) [reply docs/instructions.md]: Yeah good idea, I'll add some more info specifying that we have made `simple-remote-notify-send` ourselves. As for install instructions, I'm not so sure - there's nothing to add that isn't covered in the release installation instructions right?

**Review by miklasr — APPROVED** [review]

**Review by C-Begley — APPROVED** [review]

**Review by Andrew-Ritchie — APPROVED** [review]
Looks really good @Ashwin-MJ, I have no requests for any changes! :)

**Pull request merged**